tests/vectordb/test_zilliz_db.py

```diff
@@ -0,0 +1,89 @@
+# ruff: noqa: E501
+
+import os
+from dotenv import load_dotenv
+
+import unittest
+from unittest import mock
+from unittest.mock import patch
+
+from embedchain import App
+from embedchain.config import ZillizDBConfig
+from embedchain.vectordb.zilliz import ZillizVectorDB
+
+load_dotenv()
+
+
+# to run tests, provide the URI and TOKEN in .env file
+class TestZillizVectorDBClient(unittest.TestCase):
```

Comment:
Can you please use pytest instead of using `unittest`? We are moving towards using pytest alone. 

Moreover, seems like these tests only test the `ZillizDBConfig` and not the db class itself? Can you please add tests for that as well? You may want to mock the actual zilliz client so that we don't need the credentials for testing. See tests for other dbs for reference.